tools/nni_cmd/config_schema.py

```diff
@@ -82,6 +82,16 @@ def setPathCheck(key):
         'builtinTunerName': setChoice('builtinTunerName', 'BatchTuner', 'GridSearch', 'Random'),
         Optional('gpuNum'): setNumberRange('gpuNum', int, 0, 99999),
     },
+    'TPE': {
+        'builtinTunerName': 'TPE',
+        'classArgs': {
+            Optional('optimize_mode'): setChoice('optimize_mode', 'maximize', 'minimize'),
+            Optional('parallel_optimize'): setType('parallel_optimize', bool),
+            Optional('constant_liar_type'): setChoice('constant_liar_type', 'min', 'max', 'mean')
+        },
+        Optional('includeIntermediateResults'): setType('includeIntermediateResults', bool),
```

Comment:
@SparkSnail , is it possible not to write `Optional('includeIntermediateResults')`? instead, just write one?

Reply:
@QuanluZhang  you could discuss with him in #1326